examples/deployment-manager/autogen/multivm/README.md

```diff
@@ -1,84 +1,73 @@
+This example uses the `mpdev` tool to generate a Deployment Manager template for a Redis multiple virtual machine (VM).
+
+To generate a Deployment Manager template for your VM product, 
+see the Google Cloud Marketplace documentation for [Creating your deployment package](https://cloud.google.com/marketplace/docs/partners/vm/create-deployment-package).
+
 ## Prerequisites
 
-This example package generates a Deployment Manager template for a Redis 
-multiple VM setup. Your GCP project must contain a Redis VM image to use in the
-Deployment Manager template. The following command copies the Redis VM image
-from the
-[click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
-to your GCP project.
+Your Google Cloud project must contain a Redis VM image to use in the
+Deployment Manager template. Run the following command to copy the Redis VM image
+from the [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
+to your project:
```

Comment:
> Run the following command to copy the Redis VM image
> from the [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
> to your project:

To copy the Redis VM image from the [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy) to your project, run the following command: